libs/langchain/langchain/chat_models/jinachat.py

```diff
@@ -66,11 +66,11 @@ def _create_retry_decorator(llm: JinaChat) -> Callable[[Any], Any]:
         stop=stop_after_attempt(llm.max_retries),
         wait=wait_exponential(multiplier=1, min=min_seconds, max=max_seconds),
         retry=(
-            retry_if_exception_type(openai.error.Timeout)
-            | retry_if_exception_type(openai.error.APIError)
-            | retry_if_exception_type(openai.error.APIConnectionError)
-            | retry_if_exception_type(openai.error.RateLimitError)
-            | retry_if_exception_type(openai.error.ServiceUnavailableError)
```

Comment:
[ServiceUnavailableError](https://help.openai.com/en/articles/6897204-serviceunavailableerror) no longer exists so I am swapping it out with [APIStatusError](https://github.com/openai/openai-python/blob/17ac6779958b2b74999c634c4ea4c7b74906027a/src/openai/_exceptions.py#L72) as it serves a similar purpose